android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/RibEvents.kt

```diff
@@ -33,12 +33,24 @@ public object RibEvents {
   @JvmStatic
   public val ribActionEvents: Observable<RibActionInfo> = mutableRibDurationEvents.asObservable()
 
+  private var allowRibActionEmissions = false
+
+  /**
+   * To be called before start observing/collecting on [ribActionEvents] (usually at your earliest
+   * application point)
+   */
+  @JvmStatic
+  public fun startCapturingRibActionInfo() {
```

Comment:
Do we need a disable too? For testing?

Reply:
I agree that will be useful, will add shortly